recipes/spleaf/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "spleaf" %}
+{% set version = "2.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spleaf-{{ version }}.tar.gz
+  sha256: e89f997e5d84d627e0471e7db96cea8d26222b0e2133fa85a87fa80f59be002c
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy >=1.16
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - spleaf
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.unige.ch/jean-baptiste.delisle/spleaf
+  summary: Symmetric S+LEAF matrix.
+  license: GPL-3.0-only
```

According to the [header](https://gitlab.unige.ch/Jean-Baptiste.Delisle/spleaf/-/blob/master/spleaf/cov.py), it's actually the "or any later version" variant:
```suggestion
  license: GPL-3.0-or-later
```

---

Thanks for spotting it. I applied your suggestion.